### PR TITLE
Fix 180/loading

### PR DESCRIPTION
**Pull request by inigomarquinez**
## Description

- fix: disable submit button when form is being submitted
- fix: prevent re-submitting the form while it's being submitted
- fix: remove support for node < 16 and npm < 8
- feat: add [devtools](https://www.react-hook-form.com/dev-tools/) when in DEV mode
- chore: remove commented code in example app
- refactor: move some styles to separate files
- chore: simulate long submission in the example
- ci: add ci workflow
- ci: add scorecard workflows
- feat: upgrade to node v20
- chore: update dependencies
- fix: linter errors and warnings
- ci: rename namespace from guidesmiths to onebeyond

## Related Issue

Closes https://github.com/onebeyond/react-form-builder/issues/180

## Motivation and Context

https://github.com/onebeyond/react-form-builder/issues/180

## How Has This Been Tested?

Running the example locally and also running the existing tests

## Screenshots (if appropriate):

https://github.com/onebeyond/react-form-builder/assets/25435858/4a5326ec-8e37-4484-b272-c9a565a9baf0

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by habondavid — COMMENTED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
     "type": "git",
-    "url": "git+https://github.com/guidesmiths/react-form-builder.git"
+    "url": "git+https://github.com/onebeyond/react-form-builder.git"
   },
   "main": "dist/index.js",
   "module": "dist/index.modern.js",
   "source": "src/index.js",
   "engines": {
-    "node": ">=10"
+    "node": ">=16",
```
Comment (pabloelisseo) [inline package.json:15]: We have here node 16, maybe we should use 16 in the .nvmrc

Comment (inigomarquinez) [reply package.json:15]: 🤔

File: .nvmrc [hunk 2]
```diff
@@ -1 +1 @@
-v14
+v20
```
[the remark this answers is not in the document]
Comment (inigomarquinez) [reply .nvmrc:1]: Good point... Perhaps we should, yes. Let me reinstall with node 16 version

**Review by pabloelisseo — APPROVED** [review]
Amazing!

**Pull request merged**